resources/benchmark-runner.mjs

```diff
@@ -5,15 +5,12 @@ export class BenchmarkTestStep {
     }
 }
 
-
 class Page {
-    constructor(frame)
-    {
+    constructor(frame) {
```

Comment:
This is not right either. Curly braces should be on their own line:
https://webkit.org/code-style-guidelines/#braces-function

Reply:
So are we reverting these curly brace changes??

Reply:
As I mentioned during today's sync, we're okay with adopting K&R style curly braces everywhere.